Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -314,28 +319,28 @@ class BocList
     // Unselect currentRow
     rowBlock.SelectorControl.checked = false;
     if (isRowHighlightingEnabled)
-      $(rowBlock.Row).removeClass(BocList.TrClassNameSelected);
+      rowBlock.Row.classList.remove(BocList.TrClassNameSelected);
 
     BocList.ClearSelectAllRowsSelector (selectedRows);
   }
 
   private static SetSelectAllRowsSelectorOnDemand (selectedRows: BocList_SelectedRows): void
   {
     if (selectedRows.DataRowCount == selectedRows.Length && selectedRows.DataRowCount > 0)
-      selectedRows.SelectAllSelectorControls!.each (function (this: JQuery) { this.checked = true; }); // TODO RM-7711 - Move BocList's Row Selection logic to TypeScript class 'BocList_SelectedRows'.
+      selectedRows.SelectAllSelectorControls!.forEach (element => { element.checked = true; }); // TODO RM-7711 - Move BocList's Row Selection logic to TypeScript class 'BocList_SelectedRows'.
   }
 
   private static ClearSelectAllRowsSelector (selectedRows: BocList_SelectedRows): void
   {
-    selectedRows.SelectAllSelectorControls!.each (function (this: JQuery) { this.checked = false; }); // TODO RM-7711 - Move BocList's Row Selection logic to TypeScript class 'BocList_SelectedRows'.
+    selectedRows.SelectAllSelectorControls!.forEach (element => { element.checked = false; }); // TODO RM-7711 - Move BocList's Row Selection logic to TypeScript class 'BocList_SelectedRows'.
   }
 
   //  Event handler for the selection selectorControl in the title row.
   //  Applies the checked state of the title's selectorControl to all data rows' selectu=ion selectorControles.
   //  bocList: The BocList to which the selectorControl belongs.
   //  selectRowControlName: The name of the row selector controls.
   //  isRowHighlightingEnabled: true to enable highting of the rows
-  public static OnSelectAllSelectorControlClick (bocList: HTMLElement, selectAllSelectorControl: JQuery, isRowHighlightingEnabled: boolean): void
+  public static OnSelectAllSelectorControlClick (bocList: HTMLElement, selectAllSelectorControl: HTMLInputElement, isRowHighlightingEnabled: boolean): void
```

Comment:
Since this method is public we should introduce the selector pattern as well to stay consistent.